pkg/registry/ecr.go

```diff
@@ -107,6 +107,7 @@ func (e *ECRClient) buildEcrTags() []*ecr.Tag {
 	ecrTags := []*ecr.Tag{}
 
 	for _, t := range e.tags {
+		t := t
 		tag := ecr.Tag{Key: &t.Key, Value: &t.Value}
```

Comment:
@darkweaver87 rather do this:
```suggestion
		tag := ecr.Tag{Key: aws.String(t.Key), Value: aws.String(t.Value)}
```